todolist/app/constants/styleConst.js

```diff
@@ -0,0 +1,13 @@
+//Colors
+export const GREY = "#D6DBDF";
```

Comment:
you should have to differnt files in constants folder. One called colors.js and the other can be called dimensions.js